src/CMakeLists.txt

```diff
@@ -1,4 +1,14 @@
 
+#
+# External libaries
+#
+
+FetchContent_Declare(
+        span-lite
+        GIT_REPOSITORY https://github.com/martinmoene/span-lite.git
```

Comment:
Where is span-lite used? Do we need this dependency?

Reply:
It is not used at the moment. I thought about using the `span` feature. Thanks for pointing it out. I removed it.